ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -523,282 +316,193 @@ private void loadProperties() {
+        } catch (final IllegalArgumentException e) {
+            System.err.printf("Error: Unknown priority: %s", priorityString);
+            System.exit(1);
+        }
+        return null;
     }
-
     public WriteOptimization getWriteOptimization() {
-        return this.writeOptimization;
-    }
-
-    void setWriteOptimization(final WriteOptimization writeOptimization) {
-        this.writeOptimization = writeOptimization;
-    }
-
-    public boolean isCertificateVerification() {
-        return this.certificateVerification;
-    }
-
-    void setCertificateVerification(final boolean certificateVerification) {
-        this.certificateVerification = certificateVerification;
-    }
-
-    public boolean isHttps() {
-        return this.https;
-    }
-
-    void setHttps(final boolean https) {
-        this.https = https;
-    }
-
-    public ViewType getOutputFormat() {
-        return this.outputFormat;
-    }
-
-    void setOutputFormat(final ViewType outputFormat) {
-        this.outputFormat = outputFormat;
-    }
-
-    public String getId() {
-        return this.id;
-    }
-
-    void setId(final String id) {
-        this.id = id;
-    }
-
-    public boolean isCompleted() {
-        return this.completed;
-    }
-
-    void setCompleted(final boolean completed) {
-        this.completed = completed;
-    }
-
-    void setSync(final boolean sync) {
-        this.sync = sync;
-    }
-
-    public boolean isSync() {
-        return this.sync;
-    }
-
-    void setNumberOfFiles(final String numberOfFiles) {
-        this.numberOfFiles = numberOfFiles;
-    }
-
-    public String getNumberOfFiles() {
-        return this.numberOfFiles;
-    }
-
-    void setSizeOfFiles(final String sizeOfFiles) {
-        this.sizeOfFiles = sizeOfFiles;
-    }
-
-    public String getSizeOfFiles() {
-        return this.sizeOfFiles;
-    }
-
-    void setBufferSize(final String bufferSize) {
-        this.bufferSize = bufferSize;
-    }
-
-    public String getBufferSize() {
-        return this.bufferSize;
-    }
-
-    void setNumberOfThreads(final String numberOfThreads) {
-        this.numberOfThreads = numberOfThreads;
-    }
-
-    public String getNumberOfThreads() {
-        return this.numberOfThreads;
-    }
-
-    public boolean isIgnoreErrors() {
-        return this.ignoreErrors;
-    }
-
-    void setIgnoreErrors(final boolean ignoreErrors) {
-        this.ignoreErrors = ignoreErrors;
-    }
-
-    public boolean isFollowSymlinks() {
-        return followSymlinks;
+        final String writeOptimizationString = this.getOptionValue(WRITE_OPTIMIZATION.getLongOpt());
+        try {
+            if (writeOptimizationString == null) {
+                return null;
+            } else {
+                return WriteOptimization.valueOf(writeOptimizationString.toUpperCase());
+            }
+        } catch (final IllegalArgumentException e) {
+            System.err.printf("Error: Unknown write_optimization: %s", writeOptimizationString);
```

Comment:
This should follow the normal exception handling path and let the caller handle the printing of the exception.